rtloader/test/datadog_agent/datadog_agent.go

```diff
@@ -247,3 +249,24 @@ func obfuscateSQL(rawQuery *C.char, errResult **C.char) *C.char {
 		return nil
 	}
 }
+
+//export obfuscateSQLExecPlan
+func obfuscateSQLExecPlan(rawQuery *C.char, normalize C.bool, errResult **C.char) *C.char {
+	shouldNormalize := normalize == C.bool(true)
+	switch C.GoString(rawQuery) {
+	case "raw-json-plan":
+		if shouldNormalize {
```

Comment:
same here, why not `if bool(normalize) {` ?

Reply:
Good point, updated